android-components/components/compose/awesomebar/src/test/java/mozilla/components/compose/browser/awesomebar/internal/SuggestionFetcherTest.kt

```diff
@@ -66,4 +66,21 @@ class SuggestionFetcherTest {
 
         assertEquals(providerGroup.priority, Int.MAX_VALUE)
     }
+
+    @Test
+    fun `GIVEN a suggestion group with no suggestions THEN the group's priority should be the default one`() = runTestOnMain {
+        val provider: SuggestionProvider = mock()
+        val providerGroup = SuggestionProviderGroup(listOf(provider))
+
+        val fetcher = spy(SuggestionFetcher(listOf(providerGroup), null))
+
+        fetcher.processResultFrom(
+            group = providerGroup,
+            provider = provider,
+            suggestions = emptyList(),
+            profilerStartTime = null,
+        )
+
+        assertEquals(providerGroup.priority, -1)
```

Comment:
expected and actual objects are swapped, should be `-1, providerGroup.priority`

Reply:
Done